src/main/java/seedu/address/model/Hotel.java

```diff
@@ -438,4 +440,19 @@ public void addStay(Stay stay) {
         stayList.add(stay);
     }
 
+    /**
+     * Goes through stay list and sets
+     * the current stay for each room
+     */
+    public void updateRoomStays() {
+        for (Stay stay : stayList) {
+            for (Room room : roomList) {
+                if (stay.getRoom().equals(room)) {
+                    Room newRoom = new Room(room);
+                    newRoom.setStay(stay);
```

Comment:
Can we just modify the stay in the old object?

Reply:
I tried to do this and the list of rooms did not update in the UI when I checked in someone. If you have any idea why this happens please share with me.